schema.sql

```diff
@@ -3,3 +3,12 @@
 CREATE TABLE animals (
     name varchar(100)
 );
+
+CREATE TABLE animals ( 
+    id SERIAL PRIMARY KEY, 
+    name VARCHAR, 
```

Comment:
- [ ] The name column being of VARCHAR data type should have a limit. Example `name varchar(255),`